tools/documentation/checkDocsCurrent.sh

```diff
@@ -4,13 +4,15 @@ set -e
 
 # Generate package docs in the working directory
 echo "Document..."
+PRE_STATUS=$(git status --porcelain)
```

Comment:
Please add a comment explaining why things are done this way (for GHA)

Reply:
This was fixed by r-lib/actions. I've reverted the changes as the original code is easier to understand